Servlet/Assignment 1/src/com/servlets/DoGetInput.java

```diff
@@ -0,0 +1,50 @@
+package com.servlets;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+public class DoGetInput extends HttpServlet {
+protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+	String name = request.getParameter("name");
+	String email = request.getParameter("email");
+	String dob = request.getParameter("dob");
+	String password = request.getParameter("password");
+	String confirmPassword = request.getParameter("confirmPassword");
+
+	PrintWriter out = response.getWriter();
+
+	String error = "";
+
+	if (name == null || name == "")
```

Comment:
ACTION: We should not use {name == ""} to check an empty string. Use {name.isEmpty()} instead.